hack/jenkins/job-configs/test-linkchecker.yaml

```diff
@@ -0,0 +1,66 @@
+- job-template:
+    name: 'kubernetes-test-linkchecker'
+    description: |
+        Grab the latest from GitHub, then run hack/verify-linkcheck.sh.<br>
+        Test Owner: Build Cop
+    logrotate:
+        numToKeep: 200
+    builders:
+        - shell: './hack/verify-linkcheck.sh'
+    publishers:
+        - claim-build
+        - gcs-uploader
+        - log-parser
+        - email-ext
+        - xunit:
+            thresholds:
+                - failed:
+                    unstable: 0
+                    unstablenew: 0
+                    failure:
+                    failurenew:
+                - skipped:
+                    unstable:
+                    unstablenew:
+                    failure:
+                    failurenew:
+            types:
+                - junit:
+                    pattern: '_artifacts/**.xml'
+                    deleteoutput: false
+    scm:
+        - git:
+            url: https://github.com/kubernetes/kubernetes
+            branches:
+                - 'master'
+            browser: githubweb
+            browser-url: https://github.com/kubernetes/kubernetes
+            wipe-workspace: false
+            skip-tag: true
+    triggers:
+        - timed: 'H 0 * * *'
+    wrappers:
+        - ansicolor:
+            colormap: xterm
+        - timeout:
+            timeout: 60
```

Comment:
It takes more than 20 minutes on my local machine. And the time varies depending on how many 503 the tool gets, so I choose 60 minutes.